server/graphcool.yml

```diff
@@ -37,22 +30,21 @@ permissions:
 #   - mytoken
 
 # added by email-password template: (please uncomment)
+# functions:
+#   signup:
+#     type: resolver
+#     schema: src/email-password/signup.graphql
+#     handler:
+#       code: src/email-password/signup.ts
 
-functions:
-  signup:
-    type: resolver
-    schema: src/email-password/signup.graphql
-    handler:
-      code: src/email-password/signup.ts
-
-  authenticate:
-    type: resolver
-    schema: src/email-password/authenticate.graphql
-    handler:
-      code: src/email-password/authenticate.ts
+#   authenticate:
+#     type: resolver
+#     schema: src/email-password/authenticate.graphql
+#     handler:
+#       code: src/email-password/authenticate.ts
 
-  loggedInUser:
-    type: resolver
-    schema: src/email-password/loggedInUser.graphql
-    handler:
-      code: src/email-password/loggedInUser.ts
+#   loggedInUser:
+#     type: resolver
+#     schema: src/email-password/loggedInUser.graphql
+#     handler:
+#       code: src/email-password/loggedInUser.ts
```

Comment:
Do you want to remove these functions?

Reply:
Nope. We need these. They become uncommented in the NC-feature-Auth branch

Reply:
We need them in this branch too. Please uncomment these lines.